zul/src/archive/web/js/zul/inp/Slider.js

```diff
@@ -211,6 +211,8 @@ zul.inp.Slider = zk.$extends(zul.Widget, {
 	domClass_: function () {
 		var scls = this.$supers('domClass_', arguments),
 			isVertical = this.isVertical();
+		if (this._mode == 'knob')

```

Comment:
In theory, you don't need to modify zul widget class because we will create a new extended class to override what we want.

Reply:
yes, at first time it affect the workability, but it doesn't affect anymore, thanks.

Reply:
done